### DISCUSSION TITLE
Banner Image Details

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1326] [Support]
Can someone post the details for sizing images for the various banner iterations? 

Issue I'm running into: 

https://i.imgur.com/woRWXvQ.png

**Comment by tosteiner** [comment]
ooh, just discovering this now - I'd second this, a short guide on image dimensions would be really useful!

**Comment by amdeis** [comment]
Hoping also for image dimensions for Medium sized pubs? (preview image for large are 1200x800 I believe, but not for medium)

**Comment by meeloun** [comment]
When choosing Essay ghostwriting          [http://www.emwchinese.com/store/info?id=1](http://www.emwchinese.com/store/info?id=1)          services, you need to pay attention to multiple aspects to ensure that your choice is correct. By carefully identifying and communicating with institutions, it can help you find personalized services that truly meet your needs.
